src/autocomplete/autocomplete.js

```diff
@@ -31,6 +32,13 @@ export default ({
   const [isLoading, setIsLoading] = useState(false)
   const inputRef = useRef()
 
+  // call user-supplied onFeatures callback
+  if (typeof userOnFeatures === 'function') {
+    useEffect(() => {
+      userOnFeatures(results.features)
+    })
```

Comment:
This needs `[results]` as a second argument to `useEffect` so that it is only called if the `results` actually change.

Reply:
resolved via rebase